### PR TITLE
feat(wizard): update various text copies

### PR DESCRIPTION
**Pull request by hiaux0**
##### Proposal
image

##### Lead Details
image

##### Primary/Partner DAO
image

##### Token Details
image

### CODE REVIEW

File: src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html [hunk 1]
```diff
@@ -65,7 +65,7 @@
       </pform-input>
       <pform-input
         label="Instant Transfer Set-up"
-        label-description="Once the deal is executed on-chain, the Instant Transfer amount will be immediately sent to the other DAO’s treasury address. You can adjust how much of the total amount is set to be transferred instantly and how much will be vested (see below).">
+        label-description="Once the deal is executed on-chain, the Instant Transfer amount will be immediately sent to the other DAO’s treasury address, whereas the Vested amount will define the amount of tokens that will be released progressively throughout the total Vesting period defined in the Vesting Set-up below.">
```
Comment (dkent600) [inline src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html:68]: "Set-up" should be "setup"

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/wizards/tokenSwapDealWizard/stages/partnerDaoStage/partnerDaoStage.html [hunk 2]
```diff
@@ -36,7 +36,7 @@
         <br /><br />
         A deal gets accepted if more than half of the representatives of involved DAO's vote agree.
         <br /><br />
-        Please put the address of the Proposal Lead if they are part of the Primary DAO, and wish to partake in the voting process.
+        Note, that the Proposal Lead is not allowed to deposit funds from the UI, nor to partake in the voting process unless their address is also listed as Representative, so if you want to also fund or cast votes from the Proposal Lead address please add it here as a Representative too..
```
Comment (dkent600) [inline src/wizards/tokenSwapDealWizard/stages/partnerDaoStage/partnerDaoStage.html:39]: should be just one '.'

File: src/wizards/tokenSwapDealWizard/stages/primaryDaoStage/primaryDaoStage.html [hunk 3]
```diff
@@ -36,7 +36,7 @@
         <br /><br />
         A deal gets accepted if more than half of the representatives of involved DAO's vote agree.
         <br /><br />
-        Please put the address of the Proposal Lead if they are part of the Primary DAO, and wish to partake in the voting process.
+        Note, that the Proposal Lead is not allowed to deposit funds from the UI, nor to partake in the voting process unless their address is also listed as Representative, so if you want to also fund or cast votes from the Proposal Lead address please add it here as a Representative too..
```
Comment (dkent600) [inline src/wizards/tokenSwapDealWizard/stages/primaryDaoStage/primaryDaoStage.html:39]: "Note, that" should be "Note that"

Should be only one '.'

File: src/wizards/tokenSwapDealWizard/stages/proposalStage/proposalStage.html [hunk 4]
```diff
@@ -13,7 +13,7 @@ <h2 data-test="section-title" class="sidebarHeader heading heading3 title">Propo
           You are about to create a <strong>Token Swap Deal</strong>, which allows you to easily define the on-chain conditions of your deal, negotiate and safely exchange tokens with other DAOs.
         </p>
         <p class="sidebarText">
-          This wizard will guide you through the process. You can always come back to edit your input before you <strong>submit</strong> the deal.
+          This wizard will guide you through the process. You can always come back to edit your input before you hit the <strong>submit</strong> button at the end of the wizard.
```
Comment (dkent600) [inline src/wizards/tokenSwapDealWizard/stages/proposalStage/proposalStage.html:16]: "hit the <strong>submit</strong>" should be "click the <strong>SUBMIT</strong>"

**Review by hiaux0 — COMMENTED** [review]
.

File: src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html [hunk 5]
```diff
@@ -64,8 +64,8 @@
           view-model.ref="amountInput"></pinput-numeric>
       </pform-input>
       <pform-input
-        label="Instant Transfer Set-up"
-        label-description="Once the deal is executed on-chain, the Instant Transfer amount will be immediately sent to the other DAO’s treasury address. You can adjust how much of the total amount is set to be transferred instantly and how much will be vested (see below).">
+        label="Instant Transfer Setup"
```
[the remark this answers is not in the document]
Comment (dkent600) [reply src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html:67]: yes, thanks

File: src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html [hunk 6]
```diff
@@ -78,8 +78,8 @@
           right.bind="token.vestedTransferAmount & validate"></prange-slider>
       </pform-input>
 
-      <pform-input label="Vesting Set-up"
-        label-description="Decide how long the tokens should be vested for, and the cliff period which defines the days after which the tokens can be claimed.">
+      <pform-input label="Vesting Setup"
```
[the remark this answers is not in the document]
Comment (dkent600) [reply src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html:81]: yes

File: src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (hiaux0) [reply src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html:67]: Sry meant to express that I did the change already, and asked whether you were okay with them.

File: src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html (hunk 6 above)
[the remark this answers is not in the document]
Comment (hiaux0) [reply src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html:81]: Sry meant to express that I did the change already, and asked whether you were okay with them.

**Review by dkent600 — APPROVED** [review]

**Pull request merged**